x/upgrade/client/cli/query.go

```diff
@@ -110,3 +112,45 @@ func GetAppliedPlanCmd() *cobra.Command {
 
 	return cmd
 }
+
+// GetVersionMapCmd returns the version map from state
+func GetVersionMapCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "versionmap [optional module_name]",
+		Short: "get module version map",
+		Long: "Gets a map of module names and their respective consensus versions.\n" +
+			"Following versionmap with a specific module name will return only the\n" +
+			"version of that module.",
+		Args: cobra.MinimumNArgs(0),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientQueryContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			queryClient := types.NewQueryClient(clientCtx)
+			var params types.QueryVersionMap
+
+			if len(args) == 1 {
+				params = types.QueryVersionMap{ModuleName: args[0]}
+			} else {
+				params = types.QueryVersionMap{}
+			}
```

Comment:
```suggestion
			params = types.QueryVersionMap{ModuleName: args[0]}
```

I think we can, right?